reliability-engineering/terraform/modules/concourse-secrets-admin/iam.tf

```diff
@@ -0,0 +1,87 @@
+resource "aws_iam_role" "concourse_secrets_admin" {
+  name = "${var.deployment}-${var.concourse_team_name}-concourse-secrets-admin"
+
+  assume_role_policy = <<-EOF
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Principal": {
+          "Federated": "${module.concourse_base.aws_iam_openid_connect_provider_arn}"
+        },
+        "Action": "sts:AssumeRoleWithWebIdentity",
+        "Condition": {
+          "StringEquals": {
+            "${var.oidc_host_path}:aud": "${var.github_oauth_client_id}",
+            "aws:RequestTag/t${var.trusted_github_team_id}": "t"      // instantiated in team.tf files in tech-ops-private
+          }                                                          // and we need to supply github team id as a discrete no.
+        }                                                           // change name to team_name
+      },                                                            // VPN cidr's (IP restrictions)
```

Comment:
I don't understand these comments.

Reply:
I've removed them now, I think they were not particularly clear notes. There's stuff in Trello to cover work that still needs doing (like IP restrictions).